decentraland/unity-renderer
# fix: [BiW Editor] Scene Limits calculation is wrong on number of Materials

## What this PR includes?

### Fixes #527 
- [x] Scene limits in BIW is currently broken while counting duplicated materials.

![image](https://user-images.githubusercontent.com/64659061/120228889-52f4a580-c24c-11eb-8bfc-85a73fdf5053.png)

## How to test it?

1. Go to: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:fix/BiW-Editor-Scene-Limits-calculation-is-wrong-on-number-of-Materials&ENV=zone&ENABLE_BUILDER_IN_WORLD&position=80,154
2. Press 'K' to open the Builer In World editor.
3. Instantiate multiple assets and materials.
4. Make sure that you duplicate the same entity several times, observing the Material count does not increase for each duplicated entity.
5. Exit the scene.
6. Re-enter the editor and check the Materials counter.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
